{"name": "Получение задачи с невалидным ID", "status": "passed", "steps": [{"name": "Генерируем данные для новой задачи", "status": "passed", "attachments": [{"name": "Сгенерированные данные", "source": "3e3bd796-f2a0-406f-b66d-52094e449292-attachment.json", "type": "application/json"}], "start": 1746535620721, "stop": 1746535620721}, {"name": "Создаем задачу через API", "status": "passed", "attachments": [{"name": "Тело запроса", "source": "3a91c456-67ec-4c80-890f-52cb85431a68-attachment.json", "type": "application/json"}, {"name": "Ответ сервера", "source": "d3409519-5b27-4ae0-9797-3bae8044563b-attachment.json", "type": "application/json"}], "start": 1746535620721, "stop": 1746535621480}, {"name": "Отправляем GET запрос с некорректным ID: ", "status": "passed", "start": 1746535621480, "stop": 1746535621626}, {"name": "Проверяем код ответа и сообщение об ошибке", "status": "passed", "start": 1746535621626, "stop": 1746535621626}], "parameters": [{"name": "wrong_id", "value": "''"}], "start": 1746535620721, "stop": 1746535621627, "uuid": "ded1422a-8ad2-4d91-9e91-8c3750cf0ba0", "historyId": "3e916150ead964e7afe384aca8020f86", "testCaseId": "3e916150ead964e7afe384aca8020f86", "fullName": "tests.api.test_tasks.TestTasks#test_get_task_invalid", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Egors-MacBook-Air.local"}, {"name": "thread", "value": "21580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}